arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Iterable.kt

```diff
@@ -1290,13 +1256,17 @@ public fun <A, B> Iterable<A>.crosswalkNull(f: (A) -> B?): List<B>? =
     )
   }
 
+@Deprecated("Not being used anymore. Will be removed from the binary in 2.0.0")
 @PublishedApi
 internal val listUnit: List<Unit> =
   listOf(Unit)
 
+@Deprecated(
+  NicheAPI,
+  ReplaceWith("toList().let { l -> List(n) { l } }")
```

Comment:
Why is the replacement not the same as the implementation?

Reply:
The implementation can be updated to be the same.